.github/workflows/build-bazel.yml

```diff
@@ -71,3 +71,11 @@ jobs:
             @unordered_dense @wasmtime @xz
         - name: build
           run: bazel build //...
+        - name: build thirdparty
+          run: |
+            bazel build --config=release @ada @avro @base64 @c-ares @hdrhistogram \
+            @hwloc @krb5 @libpciaccess @libxml2 @lksctp @numactl @openssl \
+            @openssl-fips @rapidjson @roaring @seastar @snappy \
+            @unordered_dense @wasmtime @xz
+        - name: build
+          run: bazel build --config=release //...
```

Comment:
We should have a discussion about what we test in CI. Do we test fastbuild, release and debug? or just debug and release? Or sanitizers and release?

Reply:
i would say sanitizers plus release